src/spy-listener.ts

```diff
@@ -8,6 +8,10 @@ const parseDebugName = (name: string) => {
   return name.replace(/@\d+/, '');
 };
 
+const isMobxCaughtException = (value: any) => {
+  return Object.prototype.toString.call(value) === '[object Object]' && value?.constructor?.name === 'CaughtException';
```

Comment:
Codacy has a fix for the issue: [Replace `Object.prototype.toString.call(value)·===·'[object·Object]'·&&·value?.constructor?.name·===·'CaughtException'` with `(⏎····Object.prototype.toString.call(value)·===·'[object·Object]'·&&⏎····value?.constructor?.name·===·'CaughtException'⏎··)`](https://app.codacy.com/gh/kubk/mobx-log/pullRequest?prid=9947101)
```suggestion
  return (
```